mod-configuration-client/src/test/java/org/folio/rest/TestClient.java

```diff
@@ -61,17 +55,10 @@ public void setUp(TestContext context) throws IOException {
 
   }
 
-  private static void setupPostgres() throws Exception {
-    PostgresClient.setIsEmbedded(true);
-    PostgresClient.setEmbeddedPort(NetworkUtils.nextFreePort());
-    PostgresClient.getInstance(vertx).startEmbeddedPostgres();
-  }
-
   @After
   public void tearDown(TestContext context) {
     Async async = context.async();
     vertx.close(context.asyncAssertSuccess( res-> {
-      PostgresClient.stopEmbeddedPostgres();
       async.complete();
     }));
   }
```

Comment:
The tearDown method can be removed because VertxUnitRunner automatically closes vertx.